### ISSUE TITLE
Can't upvote

### ISSUE THREAD

**Issue by n0wastaken** [nborrmann/jodel_api#9]
While using a verified account, I can't upvote or downvote anything.
Everything else works.

```
>>> j.get_karma()[0]
200
>>> j.upvote('58b97e57c57d2b2300ca57a4')[0]
477
```

The first Jodel is one from their team saying that the client is outdated, maybe it's a restriction coming from this.
I tried posting and getting a little bit of karma, to see if it was a security concern, but it seems not.

**Comment by nborrmann** [comment]
Yeah, I know this doesn't work right now. 

Code `477` means invalid account signing, so maybe they have changed their request signing procedure for vote requests to prevent shenanigans. 

You could have a look at this [project](https://bitbucket.org/cfib90/ojoc) and see if voting works there. If it does, I made a mistake and will fix it.

**Comment by n0wastaken** [comment]
Hey,
Just tested it with nojoc.py, and it seems to work fine!

**Comment by nborrmann** [comment]
Alright fixed it in 57502a41ae1e011498aef210eeb289f200df98e0.

Turns out this was broken for a long time, but they didn't enforce request signing for everything, only creating accounts and posts, which happend to work correctly.
